routing/router.go

```diff
@@ -44,6 +44,10 @@ var (
 	// construct a new sphinx packet, but provides an empty set of hops for
 	// each route.
 	ErrNoRouteHopsProvided = fmt.Errorf("empty route hops provided")
+
+	// ErrRouterShuttingDown is returned if the router is in the process of
+	// shutting down.
+	ErrRouterShuttingDown = fmt.Errorf("router shutting down")
```

Comment:
Reminder: make part of `routing.Error` type?

Reply:
If we merge this PR, how is this comment going to remind you?

Reply:
Don't merge just yet 😛

Reply:
Did a sweep of the `fundingmgr` and only `routing.ErrOutdated` and `routing.ErrIgnored` is ignored there, so this should be safe to merge.